src/5.0_to_5.1/calculated-measures/getNamesOfCalculatedMeasuresToMigrate.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  ContentRecord,
+  findContentRecords,
+} from "@activeviam/activeui-sdk-5.0";
+import _map from "lodash/map";
+import { _getMetaData } from "../../_getMetaData";
+
+/**
+ * Returns the names of the calculated measures to migrate, indexed by the corresponding file ids under /ui/calculated_measures/content.
+ */
+export function getNamesOfCalculatedMeasuresToMigrate(
+  contentServer: ContentRecord,
+): string[] {
+  const calculatedMeasuresFolder =
+    contentServer.children?.ui?.children?.calculated_measures;
+
+  const { content, structure } = calculatedMeasuresFolder?.children ?? {};
+  if (!content?.children || !structure?.children) {
+    return [];
+  }
+
+  const structureRecords = findContentRecords(
+    structure,
+    Object.keys(content.children),
+  );
+
+  return _map(structureRecords, ({ pathToParentFolder }, id) => {
+    const metadata = _getMetaData(structure, pathToParentFolder, id);
```

Comment:
`getMetaData` expects the node's content to be parsed, but it's not the case here so I am using `_getMetaData` instead.
It makes it consistent with the rest of the codebase.

Reply:
It's insignificant. And I prefer to do the parsing on a per file basis, only when needed, through `_getMetaData` rather than doing it once and for all on the whole content server and then using `getMetaData` like we do in ActiveUI. It's a different use case: it's more performant in ActiveUI, but it would be less performant here.